CMakeLists.txt

```diff
@@ -66,8 +71,10 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     endif()
 endif()
 
-list(APPEND CMAKE_MODULE_PATH "${UNITS_SOURCE_DIR}/config")
-list(APPEND CMAKE_MODULE_PATH "${UNITS_SOURCE_DIR}/ThirdParty/cmake")
+list(APPEND CMAKE_MODULE_PATH "${${UNITS_CMAKE_PROJECT_NAME}_SOURCE_DIR}/config")
```

Comment:
Think you can use CMAKE_CURRENT_SOURCE_DIR here, which might be easier to follow.

Reply:
👍